docker/legacy-auth/env.template

```diff
@@ -15,11 +15,20 @@ AUTH0_SECRET=
 AUTH0_BASE_URL=https://example.fiftyone.ai
 
 # This should be set to the URI your end-users will use to connect to the API
-#  This could be the same as AUTH0_BASE_URL
+#  This could be the same as AUTH0_BASE_URL if you are using path-based routing
 FIFTYONE_API_URI=https://example-api.fiftyone.ai
 
 # This should be a MongoDB Connection String for your database
 FIFTYONE_DATABASE_URI="mongodb://username:password@mongodb-example.fiftyone.ai:27017/?authSource=admin"
+# If you are using a different MongoDB Connection String for your CAS database,
+#  set it here
+# CAS_MONGODB_URI="mongodb://username:password@mongodb-cas-example.fiftyone.ai:27017/?authSource=admin"
+
+# FIFTYONE_AUTH_SECRET is a random string used to authenticate to the CAS service
+# This can be any string you care to use generated by any mechanism you prefer.
```

Comment:
I wonder if we might want to suggest a way to create a good random string here?  For example, character length or character type (letter, number, symbol).  Seems like it is used in cryptography and thus customers could benefit from a strong string.

Reply:
There ya go.